apps/cms.c

```diff
@@ -88,9 +88,7 @@ typedef enum OPTION_choice {
 } OPTION_CHOICE;
 
 const OPTIONS cms_options[] = {
-    {OPT_HELP_STR, 1, '-', "Usage: %s [options] cert.pem...\n"},
-    {OPT_HELP_STR, 1, '-',
-        "  cert.pem... recipient certs for encryption\n"},
+    {OPT_HELP_STR, 1, '-', "Usage: %s [options] [cert...]\n"},
```

Comment:
In general to distinguish between literal and replaceable parameter names (as italics cannot be used in usage outputs) the commands use capitalized words for the replaceable texts. But that would be probably a separate PR to do and maybe even not worth it.

Reply:
I'd prefer someone do that as a separate PR if it seems worthwhile.

Reply:
That's definitely OK.